packages/ringcentral-integration/modules/ConferenceCall/index.js

```diff
@@ -796,71 +794,49 @@ export default class ConferenceCall extends RcModule {
   }
 
   @proxify
-  async onMerge({ sessionId }) {
-    const session = this._webphone._sessions.get(sessionId);
-    const isSessionOnhold = session.isOnHold().local;
-    this.setMergeParty({ toSessionId: sessionId });
-    const sessionToMergeWith = this._webphone._sessions.get(this.mergingPair.fromSessionId);
+  async mergeSession({ sessionId }) {
+    const session = find(
+      x => x.id === sessionId,
+      this._webphone.sessions
+    );
+
+    if (this._webphone.isCallRecording(session)) {
```

Comment:
remove this criteria into `mergeToConference ()` also check is existing 3-way recording